### PR TITLE
Adding Camel Language Support for Emacs

### PR DESCRIPTION
**Pull request by Navyashree0923**

### CODE REVIEW

**Comment by Navyashree0923** [comment]
I did something wrong and I wasn't able to see the changes in the previous pull request. So, I have created a new Pull Request. Please review it @apupier

File: README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+# Text Editing capabilities of Camel URI with Camel XML DSL
+
+Code completion for Camel LS with XML LS
+
+![Demo](images/camell.gif)
+
+NOTE: Completion is with Ctrl+Alt+i 
+
+## prerequisites
```
Comment (apupier) [inline README.md]:
```suggestion
# Prerequisites
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+	   http://camel.apache.org/schema/spring https://camel.apache.org/schema/spring/camel-spring.xsd">
+
+  <camelContext id="camel" xmlns="http://camel.apache.org/schema/spring">
+	<route id="a route">
+	  <from uri="timer:timerName?delay=1000&amp;exchangePattern=InvalidEnumValue"/>
+	  <to uri="direct:drink"/>
+	</route>
+  </camelContext>
+</beans>
+```
```
Comment (apupier) [inline README.md:139]: Would be nice to add something like:
Then you can enjoy editing capabilities of Camel URI. For instance by calling completion with Ctrl+Alt+i in the uri attribute value.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+Opening the `lsp-camel.el` file
+
+Evaluating it by calling `M-x eval-buffer`
+
+Loading Camel LS by calling: `M-x lsp-install-server` and choosing `camells`
+
+Opening a `camel.xml` file with this kind of content:
+
+```lisp
```
Comment (apupier) [inline README.md]:
```suggestion
```xml
```

this one is xml code, not lisp

File: README.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
+## This is a client implementation of the Apache Camel Language Server Protocol for Emacs
+Link for Apache Camel Language Server 
+https://github.com/camel-tooling/camel-language-server/
+
+# Text Editing capabilities of Camel URI with Camel XML DSL
+
+Code completion for Camel LS with XML LS
```
Comment (apupier) [inline README.md]:
```suggestion
## This is a client implementation of the Apache Camel Language Server Protocol for Emacs
Link for Apache Camel Language Server 


# Text Editing capabilities of Camel URI with Camel XML DSL

For instance, code completion for Camel XML Dsl. The capabilities are based on the [Camel Language Server](https://github.com/camel-tooling/camel-language-server/).
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+![Demo](images/camell.gif)
+
+NOTE: Completion is with Ctrl+Alt+i 
+
+## prerequisites
+Java 11+ is required to be present on the System Path
+
+## How to configure Emacs
```
Comment (apupier) [inline README.md]:
```suggestion
# How to configure Emacs
```

**Review by apupier — CHANGES_REQUESTED** [review]

**Comment by Navyashree0923** [comment]
I have updated as per the comments, please review it.

**Review by apupier — APPROVED** [review]

**Pull request merged**